src/main/webapp/style.css

```diff
@@ -0,0 +1,68 @@
+#content {
+  margin-left: auto;
+  margin-right: auto;
+  width: 900px;
+}
+
+@font-face {
+   font-family: 'robotoRegular';
+   src: url(fonts/Roboto-Regular.ttf);
+}
+
+@font-face {
+   font-family: 'robotoLight';
+   src: url(fonts/Roboto-Light.ttf);
+}
+
+@font-face {
+   font-family: 'robotoThin';
+   src: url(fonts/Roboto-Thin.ttf);
+}
+
+div {
```

Comment:
The css selector is too generic for the stylings. Considering using an id instead.

Reply:
Oh good catch I'll change that so it's more clear